Configure

```diff
@@ -1193,7 +1193,9 @@ else               { $no_user_defines=1;    }
 
 # ALL MODIFICATIONS TO %config and %target MUST BE DONE FROM HERE ON
 
-unless ($disabled{engine}) {
+if ($disabled{engine}) {
+    $disabled{afalg} = "no-engine";
+} else {
```

Comment:
And.. it is also currently not possible to do 'no-afalg', i.e., you can only disable afalg via engine. Is this WAI?